### ISSUE TITLE
Machine-parseable GPDB version function

### ISSUE THREAD

**Issue by ascherbaum-pivotal** [greenplum-db/gpdb#1169]
There is no function which returns just the GPDB version number, something like:

```
SELECT gp_version();
```

**Comment by danielgustafsson** [comment]
On that note, shouldn't the CLI utils actually report the GPDB version as well as the underlying postgres version and not just the latter?
```bash
$ postmaster --version
postgres (Greenplum Database) 8.3.23
```

**Comment by Eulerizeit** [comment]
@danielgustafsson yes, yes it should. Or only report GPDB version since the Postgres version can be derived from the gpdb version but the opposite is not true.

**Comment by larham** [comment]
+1 for a function that returns some version that is easy to parse and relatively foolproof. We need it for utilities.

**Comment by danielgustafsson** [comment]
How about the proposal in PR #1663? Does that solve the problem?

**Comment by ashwinstar** [comment]
I believe this was fixed by #1663, if not feel free to open back.
